src/e2e/package.json

```diff
@@ -8,20 +8,20 @@
   },
   "private": true,
   "dependencies": {
-    "@cucumber/cucumber": "^11.1.1",
-    "@types/uniqid": "^5.3.4",
-    "cross-fetch": "^4.0.0",
-    "pdf-parse": "^1.1.1",
-    "uniqid": "^5.4.0",
-    "zod": "^3.24.1"
+    "@cucumber/cucumber": "11.1.1",
```

Comment:
Pinning these looks good to me but maybe @infonl-marcel and @xiduzo should have a look too.